app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java

```diff
@@ -43,88 +44,118 @@
+    @ColumnInfo
     public String lastname;
-    @Column
+    @ColumnInfo
     public boolean isCheckedIn;
-    @Column
+    @ColumnInfo
     public String state;
-    @Column
+    @ColumnInfo
     public String address;
-    @Column
+    @ColumnInfo
     public String pdfUrl;
-    @Column
+    @ColumnInfo
     public String country;
-    @Column
+    @ColumnInfo
     public String email;
 
     @Relationship("ticket")
-    @ForeignKey(onDelete = ForeignKeyAction.CASCADE)
+    @ForeignKey(entity = Ticket.class, parentColumns = "id", childColumns = "ticket", onDelete = ForeignKey.CASCADE)
     public Ticket ticket;
 
     @Relationship("order")
-    @ForeignKey(onDelete = ForeignKeyAction.CASCADE, saveForeignKeyModel = true)
+    @ForeignKey(entity = Order.class, parentColumns = "id", childColumns = "order", onDelete = ForeignKey.CASCADE)
     public Order order;
 
     // To associate attendees and event
     @Relationship("event")
-    @ForeignKey(stubbedRelationship = true, onDelete = ForeignKeyAction.CASCADE)
+    @ForeignKey(entity = Event.class, parentColumns = "id", childColumns = "event", onDelete = ForeignKey.CASCADE)
     public Event event;
 
     // Migration 7 additions
 
-    @Column
+    @ColumnInfo
     public String blog;
-    @Column
+    @ColumnInfo
     public String homeAddress;
-    @Column
+    @ColumnInfo
     public String workAddress;
-    @Column
+    @ColumnInfo
     public String jobTitle;
-    @Column
+    @ColumnInfo
     public String taxBusinessInfo;
-    @Column
+    @ColumnInfo
     public String phone;
-    @Column
+    @ColumnInfo
     public String gender;
-    @Column
+    @ColumnInfo
     public String company;
-    @Column
+    @ColumnInfo
     public String workPhone;
-    @Column
+    @ColumnInfo
     public String birthDate;
-    @Column
+    @ColumnInfo
     public String twitter;
-    @Column
+    @ColumnInfo
     public String facebook;
-    @Column
+    @ColumnInfo
     public String github;
-    @Column
+    @ColumnInfo
     public String website;
-    @Column
+    @ColumnInfo
     public String shippingAddress;
-    @Column
+    @ColumnInfo
     public String billingAddress;
 
     // Migration 8 additions
 
-    @Column
+    @ColumnInfo
     public String checkinTimes;
 
     // Migration 16 additions
 
-    @Column
+    @ColumnInfo
     public String checkoutTimes;
 
     // Non model entities
 
     @JsonIgnore
-    @Column
+    @ColumnInfo
     public boolean checking;
 
     public Attendee() { }
+
+    @Override
+    public AttendeeViewHolder getViewHolder(View v) {
+        return null;
+    }
+
+    @Override
+    public String getHeader() {
+        return null;
+    }
+
+    @Override
+    public long getHeaderId() {
+        return 0;
+    }
+
+    @Override
+    public int getType() {
+        return 0;
+    }
+
+    @Override
+    public int getLayoutRes() {
+        return 0;
+    }
+
+    @Override
+    public int compareTo(@NonNull Attendee attendee) {
+        return 0;
+    }
```

Comment:
These shouldn't be required

Reply:
@iamareebjamal As the class implements Comparable, Android Studio is asking to implement these functions mandatorly

Reply:
It did not ask before so how come is it now?

Reply:
And your implementations are completely wrong, so it'll only cause bugs

I don't think you have followed README and applied the kombok plugin

Reply:
When I changed @Table of DBFlow to @Entity of Room, this implementation error arised.

Reply:
I'll have a look on how can I remove them, if they aren't necessary

Reply:
An annotation change of Table to Entity can't cause this